com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+
+## Tonemapping for HDR Output
+
+When the *Use display in HDR mode* option is set in the player settings, all the HDR related options appear in the  [Tonemapping](Post-Processing-Tonemapping.md) volume component. The available options depend on the Tonemapping mode selected.
+
+Unlike tonemapping for LDR displays, the tonemapping for HDR screens require to take into account the capabilities of the device. In particular it is important to adapt to three values:
+
+- Paper white value: determines the brightness value of a paper-white surface. In practice this will determines the overall screen brightness and what brightness the UI will map to. This latter point is important as usually unlit UI is rendered assuming that a value of 1 corresponds to a white color; this assumption is not true when it comes to HDR, so HDRP uses the paper white to tune the UI so that white UI will map to a white value on screen.
+- Minimum Brightness: the minimum brightness that the display can display.
+- Maximum Brightness: the maximum brightness that the display can display before saturating the values.
+
+All the above values are in nits (candela per square meters).
+
+While it is possible to detect the above three values from the screen outputting your content, it is possible that the values communicated by the device are going to be inaccurate. For this reason, we suggest to implement a calibration menu for your application.
+
+Specifically, detected Paper white values are often going to produce dimmer results than what the LDR content will produce, the reason is that normally TV boost brightness values when displaying LDR content, but respect the actual values when outputting HDR contents.
+
+#### Neutral
+
+![HDR-Output-Neutral](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-Neutral.png)
+
+| **Property**                         | **Description**                                              |
+| ------------------------------------ | ------------------------------------------------------------ |
+| **Neutral HDR Range Reduction Mode** | The tonemapping curve used for the Neutral tonemapper. The options are:<br />- BT2390: Uses a curve defined by the BT2390 broadcasting recommendations. (Default)<br />- Reinhard: A very simple tonemapping operator.<br /><br />This option is available only when Additional Properties are displayed. |
+| **Hue Shift Amount**                 | Determines how much hue preservation is desired. When the value is 0, the tonemapper will try to preserve the hue of the content as much as possible, tonemapping only the luminance. However, some content might have been authored assuming that hue-shift will happen when all color channels are tonemapped independently. To recover the hue-shift behaviour this slider can be moved toward See note below for more information. |
```

Comment:
(just the Description)
A value that determines how much the Player can change the hue of your content. When you set this value to 0, the tone mapper tries to preserve the hue of the original content as much as possible, and only tone maps [luminance](https://en.wikipedia.org/wiki/Luminance). When you adjust this value, take into account content that relies on hue shifts that high brightness produces, such as flame effects, as illustrated in the image below. The flame on the left has a **Hue Shift Amount** of 0; the flame on the right has **Hue Shift Amount** of 1. "

![](Images/HDR-Output-HueShift.png)

Reply:
> A value that determines how much the Player can change the hue of your content

Not player specific. 

For the rest I am good, but is it good to reference an image of the docs in the param description?

Reply:
Do you mean include a screenshot of the menu? @FrancescoC-unity

Reply:
No I mean the flame thing

Reply:
Ah ok. Well, this isn't API doc, so I don't see why not. Would be best to make the images not to big, though.